app/models/levelling_up_premium_payments/eligibility.rb

```diff
@@ -96,8 +113,8 @@ def reset_dependent_answers(reset_attrs = [])
       end
     end
 
-    def eligible_none_of_the_above?
-      itt_subject_none_of_the_above?
+    def has_indicated_a_bad_itt_subject?
+      !eligible_itt_subject.nil? and !eligible_itt_subject.to_sym.in?([:chemistry, :computing, :mathematics, :physics])
```

Comment:
Suggestion: I think this list should come from JourneySubjectEligibilityChecker, being a source of truth.